src/apic-extension/azext_apic_extension/aaz/latest/apic/service/_create.py

```diff
@@ -284,10 +284,6 @@ def _build_schema_on_200_201(cls):
             )
 
             properties = cls._schema_on_200_201.properties
-            properties.data_api_hostname = AAZStrType(
```

Comment:
I believe we still need this in all of the service commands

Reply:
Searched the code and didn't find usage of `data_api_hostname`, can you help point out where it's used?

Reply:
Added the properties back